contracts/base/ModuleManager.sol

```diff
@@ -120,10 +120,10 @@ contract ModuleManager is SelfAuthorized, Executor {
         address currentModule = modules[start];
         while (currentModule != address(0x0) && currentModule != SENTINEL_MODULES && moduleCount < pageSize) {
             array[moduleCount] = currentModule;
+            next = currentModule;
```

Comment:
Instead of writing this on every loop I would just set `next` to `array[moduleCount-1]` at the end.

Reply:
Please check it now